src/naming_conventions.py

```diff
@@ -99,12 +95,7 @@ def convert_from_list(self, list_to_convert: list[str]) -> str:
             list_to_convert = a lowercased list of strings
         Returns:    A single string of snake case naming convention
         """
-        for word in list_to_convert:
-            if not word.islower():
-                raise Exception(
-                    "String wasn't lowercase for some reason. Something has gone wrong."
-                )
-        return "_".join(str(word) for word in list_to_convert)
```

Comment:
no need to case to str here. Type hints in the method's arguments should be adhered to